css/obBase.css

```diff
@@ -134,6 +134,10 @@ form {
 #sideBar {
   position: absolute;
   right: 0;
+  top: 50px;
+  width:40px;
+  background-color: #252525;
+  height: 100%;
```

Comment:
height:100% does seem to work here but I'm not particular.

Yeah I just started to play with it and just paused so we can discuss tomorrow.